postgres/changelog.d/18358.added

```diff
@@ -0,0 +1 @@
+Propagate agent host tags for postgres integration
```

Comment:
can you add more context to the changelog because we already have a changelog to propagate agent host tags from config file? want to highlight the difference

Reply:
Good point - I added more, let me know what you think. I also updated `conf.yaml.example`